### PR TITLE
[FSPE-7262] Added support for collapsed search input to header component

### PR DESCRIPTION
**Pull request by mattnickles**
https://ctl.atlassian.net/browse/FSPE-7262

### CODE REVIEW

**Comment by DiegoSanjuan** [comment]
Jenkins, test this please.

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://5c38964d02b8ae26ed8358a4759f21bd7387a050-chi-pr.enterprise.ctl.io" target="_blank"> https://5c38964d02b8ae26ed8358a4759f21bd7387a050-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1765/ for more details.

File: src/website/views/components/header/_examples.pug [hunk 1]
```diff
@@ -739,6 +739,94 @@ h3 Header With Title and Buttons
...
+                <div class="chi-button__content">
+                  <i class="chi-icon -sm--2 icon-menu"></i>
+                </div>
+              </button>
+            </div>
+          </div>
+          <div class="chi-header__end">
+            <div class="-d--none -d-lg--flex">
+              <form>
+                <div class="chi-search -collapsed -mx--1">
```
Comment (jllr) [inline src/website/views/components/header/_examples.pug:796]:
```suggestion
                <div class="chi-search -collapsed -mr--1">
```

**Comment by jllr** [comment]
@mattnickles collapsed search input is not rendering as expected in IE11

Screen Shot 2021-02-17 at 1 03 47 PM

**Comment by allyourbotsarebelongtous** [comment]
Your CHI Pull Request website is available for review at <a href="https://a73ca0c9a8f7ca046f56ddff92fe1e1531fcfe13-chi-pr.enterprise.ctl.io" target="_blank"> https://a73ca0c9a8f7ca046f56ddff92fe1e1531fcfe13-chi-pr.enterprise.ctl.io </a>

**Comment by allyourbotsarebelongtous** [comment]
Build finished SUCCESSFULLY. See http://jenkins.elasticbox.com/job/chi_pull_request/1767/ for more details.

**Review by jllr — APPROVED** [review]

**Comment by allyourbotsarebelongtous** [comment]
The following instances are being terminated: https://ops.elasticbox.com/services/instances/i-wre4s5, https://ops.elasticbox.com/services/instances/i-22gz28

**Pull request merged**